utils/httpfy/httpfy.go

```diff
@@ -23,34 +23,59 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"path"
 	"time"
 )
 
+type singleFileHandler struct {
+	name string
+}
+
+func (sf *singleFileHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// since the URL here will likely not specify a file name, pass the file name in the header
+	w.Header().Add("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", path.Base(sf.name)))
+	http.ServeFile(w, r, sf.name)
+}
+
+func serveSingleFile(filePath string) http.Handler {
```

Comment:
I really like that :+1: just serve what we have to. So even if the base image would not be deleted from the shared volume it is not reachable 😉